packages/salesforcedx-vscode-lwc/test/vscode-integration/intellisense.test.ts

```diff
@@ -8,217 +8,264 @@
+      doc = await workspace.openTextDocument(docUri);
+      editor = await window.showTextDocument(doc);
+      startPosition = new Position(5, 0);
+      endPosition = new Position(5,0);
+    });
+
+    afterEach(() => {
+      try {
+        selection = new Selection(startPosition, endPosition);
+        // We need to clear the editor's line or the input text will change and test will fail.
+        editor.edit(editBuilder => {
+          editBuilder.delete(selection)
+        });
+      } catch (e) {
+        throw e;
+      }
+    });
+
+    it('LWC Extension Activation', async function() {
+      expect(lwcExtension.isActive);
     });
 
-    try {
-      await testCompletion(docUri, endPosition, {
-        items: [
+    /**
+     * Test that lwc markup intellisense includes standard lwc tags and custom lwc tags
+     */
+    it('LWC Markup Intellisense', async () => {
+      // We have to have some text or we'll just get generic completions
+      text = '<c-';
+      endPosition = new Position(
+        startPosition.line,
+        startPosition.character + text.length
+      );
+      await editor.edit(editBuilder => {
+        editBuilder.insert(startPosition, text);
+      });
+      const items = [
           {
-            label: 'lightning/uiListApi',
-            kind: vscode.CompletionItemKind.Module
+            label: 'lightning-accordion',
+            kind: CompletionItemKind.Property
           },
           {
-            label: 'lightning/uiRecordApi',
-            kind: vscode.CompletionItemKind.Module
+            label: 'c-hello-binding',
+            kind: CompletionItemKind.Property
           }
         ]
-      });
-    } catch (error) {
-      throw error;
-    }
+
+      try {
+        // NOTE: Because the completion providers always returns all possible results and then VSCode
+        // does the filtering based on what is typed, we have no good way of testing what vscode is
+        // actually displaying to the user based on what we typed
+        // TODO - investigate why this only happens on markup
+        await testCompletion(docUri, endPosition, items);
+      } catch (error) {
+        throw error;
+      }
+    });
   });
 });
 
+
 async function testCompletion(
-  docUri: vscode.Uri,
-  position: vscode.Position,
-  expectedCompletionList: vscode.CompletionList
+  docUri: Uri,
+  position: Position,
+  expectedCompletionList: CompletionItem[]
 ) {
   // Simulate triggering a completion
-  const actualCompletionList = (await vscode.commands.executeCommand(
+  let actualCompletionList = (((await commands.executeCommand(
     'vscode.executeCompletionItemProvider',
     docUri,
     position
-  )) as vscode.CompletionList;
+  )) as CompletionList).items);
+
+  actualCompletionList.sort();
+  expectedCompletionList.sort();
 
-  expectedCompletionList.items.forEach(expectedItem => {
-    const actualItem = actualCompletionList.items.find(obj => {
+  expectedCompletionList.map(expectedItem => {
```

Comment:
Probably shouldn't use map here b/c we don't need to create a new altered array. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map#when_not_to_use_map

I'd go with for...of https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of